patches/master_patch.patch

```diff
@@ -1495,7 +1498,7 @@ diff --git a/content/browser/web_contents/web_drag_dest_mac.mm b/content/browser
 index ff2c964a68d0b21d46a3b88b2ea6c3ff4d9d0a07..af043f6d204ee90ec54bc7df401a3bd6519b8541 100644
 --- a/content/browser/web_contents/web_drag_dest_mac.mm
 +++ b/content/browser/web_contents/web_drag_dest_mac.mm
-@@ -347,6 +347,12 @@ GetRenderWidgetHostAtPoint:(const NSPoint&)viewPoint
+@@ -347,6 +347,12 @@ - (void)setDragStartTrackersForProcess:(int)processID {
```

Comment:
looks like there are some changes to the patch file that shouldn't be part of this PR